tools/integration_tests/log_content/big_file_upload_log_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2024 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package log_content
+
+import (
+	"fmt"
+	"math"
+	"testing"
+
+	"github.com/googlecloudplatform/gcsfuse/v2/tools/integration_tests/util/setup"
+)
+
+const (
+	BigFileSize            int64 = 50 * MiB
+	DirForBigFileUploadLog       = "dirForiBigFileUploadLog"
+)
+
+func TestBigFileUploadLog(t *testing.T) {
+	fileSize := BigFileSize
```

Comment:
why do you need fileSize variable?

Reply:
I created it because I needed it later while calculting no. of chunks. I have removed it now, and have defaulted to directly using BigFileSize.